DIPs/DIP-17.md

```diff
@@ -0,0 +1,73 @@
+---
+DIP: DIP-17
+Title: Hosting DevCon agenda and document sharing on Swarm
+Status: Draft
+Themes: Freeform, Website, Agenda
+Tags: Software, Event Operations, Attendee Experience
+Authors: Costanza Gallo <costanza@ethswarm.org>
+Resources Required: Operation support
+Discussion: https://forum.devcon.org/t/hosting-the-agenda-and-the-documents-on-swarm/342
+Created: 2022-06-19
+---
+
+## Summary of Proposal
+
+__Hosting the agenda of DevCon on Swarm, and using Swarm for document sharing__
+
+## Abstract
+Swarm is a decentralized storage and communication system that was incubated by the Ethereum Foundation.
+
+This DIP has one proposal:
+
+To host the website with the agenda of DevCon on Swarm.
+
+(Please note that initially in the forum we discussed two proposals, but upon further consideration we decide to split them into two for better clarity.)
+
+## Motivation & Rationale
+__Implementing this DIP would make DevCon’s website hosting decentralized and would make DevCon’s digital efforts aligned with its principles and values.__
+
+## Implementation
+- Has any part of this proposal been implemented at other events? If so, please describe how it went.
+Not yet. We are planning to reach out to more events that happen before to implement this solution. This DIP will be updated accordingly. 
+However, the solution has been implemented on other situations like blogposts: 
+http://fairdatasociety.bzz.link/
+https://fairdataprotocol.bzz.link/
```

Comment:
the first one 404s for me - the second one works

Reply:
![image](https://user-images.githubusercontent.com/111600/183120940-879afb8e-10a8-4710-84c9-31edfffd9539.png)

Reply:
maybe we should not use the 404ing one as (the first) example then